asm/istio/options/cni-managed.yaml

```diff
@@ -12,6 +12,116 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@kyessenov I am getting warning when applying:
```
Warning: apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```

I saw some discussion about this and I assume this is expected?

Reply:
if all other crds use v1beta1 then we will do en-mass migration. Since this is syncd internally changing it here will require another sync.

Reply:
This is tracked internally. You can disregard the warning since it's g3 wide.